src/main/java/io/github/guacsec/trustifyda/providers/javascript/model/Manifest.java

```diff
@@ -70,12 +70,16 @@ private Set<String> loadDependencies(JsonNode content) {
   private Set<String> loadIgnored(JsonNode content) {
     var names = new HashSet<String>();
     if (content != null) {
-      var ignore = (ArrayNode) content.get("exhortignore");
-      if (ignore == null || ignore.isEmpty()) {
-        return Collections.emptySet();
-      }
+      processIgnoreArray(content, "exhortignore", names);
+      processIgnoreArray(content, "trustify-da-ignore", names);
```

Comment:
If the new value is set, ignore the old value to enforce migration. Whoever sets the new value is only because it is being migrated.